src/common/mfu_daos.c

```diff
@@ -1041,6 +1063,20 @@ int daos_connect(
              * create a POSIX container. Otherwise, create a container of
              * the same type as the source. */
             if (!have_src_cont || (da->src_cont_type == DAOS_PROP_CO_LAYOUT_POSIX)) {
+#ifdef HDF5_SUPPORT
+                /* read container properties in hdf5 file when moving data
+                 * back to DAOS if the preserve option has been set */
+                if (preserve) {
+                    /* TODO: should always be called daos_metadata.hdf5? */
+                    daos_cont_layout_t ctype = DAOS_PROP_CO_LAYOUT_POSIX;
+                    printf("Reading metadata file: %s\n", filename);
+                    rc = cont_deserialize_all_props(&hdf5, props, entry, &ctype);
```

Comment:
@dsikich What is the purpose of passing `entry` here? It doesn't seem to be used outside of `cont_deserialize_all_props`

Reply:
@daltonbohning hmm.. I will have to take a look but I just remember the function required passing it in

Reply:
@daltonbohning I removed it since it didn't need to be passed around